march_hardware_interface/launch/testsetup.launch

```diff
@@ -19,7 +19,8 @@
           args="
                 /march/controller/joint_state
                 /march/controller/temperature_sensor
-                /march/controller/trajectory
+                /march/controller/trajectory_position
+                /march/controller/trajectory_effort
```

Comment:
Why both? Can you use both at the same time, or does one need to be commented?

Reply:
See comment above